src/app/ApplicationTemplate.Shared/Presentation/Extensions/ValidatorExtensions.cs

```diff
@@ -71,5 +72,60 @@ public static class ValidatorExtensions
 				return result;
 			});
 		}
+
+		/// <summary>
+		/// Returns true if the string is a valid email address.
+		/// </summary>
+		/// <typeparam name="T">email</typeparam>
+		/// <param name="ruleBuilder"> Rule builder</param>
+		/// <returns>Validation result.</returns>
+		public static IRuleBuilderOptions<T, string> IsValidEmail<T>(this IRuleBuilder<T, string> ruleBuilder)
```

Comment:
We know the rule isn't perfect but we want the scaffolding code to remain simple. Projects can put the rules they want.